src/common/camera_control.c

```diff
@@ -824,8 +841,67 @@ static void dt_camctl_detect_cameras(const dt_camctl_t *c)
     } while(citem && (citem = g_list_next(citem)) != NULL);
   }
 
+  /* check c->locked_cameras in available_cameras */
+  if(c->locked_cameras && g_list_length(c->locked_cameras) > 0)
```

Comment:
i think `g_list_length > 0` is pointless if `c->locked_cameras` is not null, since non-null list is always at least 1 item long, right?

Reply:
ok